src/js/utils/UserSettings.js

```diff
@@ -39,7 +40,8 @@
         maintainRatio : true,
         resizeContent : false,
         origin : 'TOPLEFT'
-      }
+      },
+      COLOR_FORMAT: 0,
```

Comment:
I see, you're right this is how the grid select is working.

You don't have to use integer values in the input, you can use value="hex" and value="rgb".
Then you will directly get this string as evt.target.value.

This should do the trick but I haven't tested :) Let me know if you have any trouble with this, will be happy to help!